google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/LoadConfiguration.java

```diff
@@ -70,6 +70,17 @@
 
 
     /**
+     * Sets the string that represents a null value in a CSV file. For example, if you specify "\N",
+     * BigQuery interprets "\N" as a null value when loading a CSV file. The default value is the
+     * empty string. If you set this property to a custom value, BigQuery throws an error if an
+     * empty string is present for all data types except for {@code STRING} and {@code BYTE}. For
+     * {@code STRING} and {@code BYTE} columns, BigQuery interprets the empty string as an empty
+     * value.
+     */
+    Builder setNullMarker(String nullMarker);
```

Comment:
I thought so, too --- however, BigQuery added the null marker field to their [`JobConfigurationLoad`](https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/java/latest/com/google/api/services/bigquery/model/JobConfigurationLoad.html#getNullMarker()) but not to their [`CsvOptions`](https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/java/latest/com/google/api/services/bigquery/model/CsvOptions.html). It's possible this was an oversight, but I'm inclined to leave ours consistent with theirs to avoid confusion.